From 286da50d846e9da8e1c03f865bd361c9ef57d315 Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Thu, 26 Sep 2024 11:55:55 +0000
Subject: [PATCH 082/114] UBUNTU: [Config] Update for hifive-premier-p550 board

Changed sdhci from builtin to modular for eMMC and SD Card boot
Add Imagination GPU configs
Changed configs for ES8388 codec and PWM
Changed config FRAME_WARN value to 2048 to fix compilation warnings

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
---
 debian/config/annotations | 86 +++++++++++++++++++--------------------
 1 file changed, 41 insertions(+), 45 deletions(-)

diff --git a/debian/config/annotations b/debian/config/annotations
index 76581116f2b8..d8c0e7efc817 100644
--- a/debian/config/annotations
+++ b/debian/config/annotations
@@ -102,12 +102,6 @@ CONFIG_DEBUG_FS                                 note<'required debug option'>
 CONFIG_DEBUG_INFO                               policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_INFO                               note<'required for debug packages'>
 
-CONFIG_DEBUG_INFO_BTF                           policy<{'riscv64': 'y'}>
-CONFIG_DEBUG_INFO_BTF                           note<'Needs newer pahole for armhf'>
-
-CONFIG_DEBUG_INFO_SPLIT                         policy<{'riscv64': 'n'}>
-CONFIG_DEBUG_INFO_SPLIT                         note<'LP: #1413664'>
-
 CONFIG_DEBUG_KERNEL                             policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_KERNEL                             note<'required debug option'>
 
@@ -243,8 +237,8 @@ CONFIG_KFENCE_STATIC_KEYS                       note<'LP: #1948038'>
 CONFIG_KGDB_SERIAL_CONSOLE                      policy<{'riscv64': 'y'}>
 CONFIG_KGDB_SERIAL_CONSOLE                      note<'early debugging support'>
 
-CONFIG_KVM                                      policy<{'riscv64': 'm'}>
-CONFIG_KVM                                      note<'LP: #1532886'>
+CONFIG_KVM                                      policy<{'riscv64': 'y'}>
+CONFIG_KVM                                      note<'TODO: update note'>
 
 CONFIG_LATENCYTOP                               policy<{'riscv64': 'y'}>
 CONFIG_LATENCYTOP                               note<'LP: #2051342'>
@@ -267,11 +261,11 @@ CONFIG_MFD_TPS65217                             note<'boot essential on AM335x'>
 CONFIG_MMC_BLOCK                                policy<{'riscv64': 'y'}>
 CONFIG_MMC_BLOCK                                note<'boot essential on armhf/arm64'>
 
-CONFIG_MMC_SDHCI                                policy<{'riscv64': 'm'}>
-CONFIG_MMC_SDHCI                                note<'installation essential on omap/highbank'>
+CONFIG_MMC_SDHCI                                policy<{'riscv64': 'y'}>
+CONFIG_MMC_SDHCI                                note<'TODO: update note'>
 
-CONFIG_MMC_SDHCI_PLTFM                          policy<{'riscv64': 'm'}>
-CONFIG_MMC_SDHCI_PLTFM                          note<'boot essential on highbank'>
+CONFIG_MMC_SDHCI_PLTFM                          policy<{'riscv64': 'y'}>
+CONFIG_MMC_SDHCI_PLTFM                          note<'TODO: update note'>
 
 CONFIG_MODULE_COMPRESS_ZSTD                     policy<{'riscv64': 'y'}>
 CONFIG_MODULE_COMPRESS_ZSTD                     note<'LP: #2028568'>
@@ -892,10 +886,9 @@ CONFIG_ASYNC_RAID6_TEST                         policy<{'riscv64': 'n'}>
 CONFIG_ASYNC_TX_DMA                             policy<{'riscv64': 'y'}>
 CONFIG_ASYNC_XOR                                policy<{'riscv64': 'm'}>
 CONFIG_AS_HAS_INSN                              policy<{'riscv64': 'y'}>
-CONFIG_AS_HAS_NON_CONST_LEB128                  policy<{'riscv64': 'y'}>
 CONFIG_AS_HAS_OPTION_ARCH                       policy<{'riscv64': 'y'}>
 CONFIG_AS_IS_GNU                                policy<{'riscv64': 'y'}>
-CONFIG_AS_VERSION                               policy<{'riscv64': '24100'}>
+CONFIG_AS_VERSION                               policy<{'riscv64': '24000'}>
 CONFIG_AT76C50X_USB                             policy<{'riscv64': 'm'}>
 CONFIG_AT803X_PHY                               policy<{'riscv64': 'm'}>
 CONFIG_ATA                                      policy<{'riscv64': 'y'}>
@@ -1267,7 +1260,6 @@ CONFIG_BOSCH_BNO055_I2C                         policy<{'riscv64': 'm'}>
 CONFIG_BOSCH_BNO055_SERIAL                      policy<{'riscv64': 'm'}>
 CONFIG_BPF                                      policy<{'riscv64': 'y'}>
 CONFIG_BPFILTER                                 policy<{'riscv64': 'y'}>
-CONFIG_BPFILTER_UMH                             policy<{'riscv64': 'm'}>
 CONFIG_BPF_EVENTS                               policy<{'riscv64': 'y'}>
 CONFIG_BPF_JIT                                  policy<{'riscv64': 'y'}>
 CONFIG_BPF_JIT_ALWAYS_ON                        policy<{'riscv64': 'y'}>
@@ -1479,8 +1471,6 @@ CONFIG_CB710_DEBUG                              policy<{'riscv64': 'n'}>
 CONFIG_CB710_DEBUG_ASSUMPTIONS                  policy<{'riscv64': 'y'}>
 CONFIG_CC10001_ADC                              policy<{'riscv64': 'm'}>
 CONFIG_CCS811                                   policy<{'riscv64': 'm'}>
-CONFIG_CC_CAN_LINK                              policy<{'riscv64': 'y'}>
-CONFIG_CC_CAN_LINK_STATIC                       policy<{'riscv64': 'y'}>
 CONFIG_CC_HAS_ASM_GOTO_OUTPUT                   policy<{'riscv64': 'y'}>
 CONFIG_CC_HAS_ASM_GOTO_TIED_OUTPUT              policy<{'riscv64': 'y'}>
 CONFIG_CC_HAS_ASM_INLINE                        policy<{'riscv64': 'y'}>
@@ -1499,12 +1489,12 @@ CONFIG_CC_IS_GCC                                policy<{'riscv64': 'y'}>
 CONFIG_CC_NO_ARRAY_BOUNDS                       policy<{'riscv64': 'y'}>
 CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE              policy<{'riscv64': 'y'}>
 CONFIG_CC_OPTIMIZE_FOR_SIZE                     policy<{'riscv64': 'n'}>
-CONFIG_CC_VERSION_TEXT                          policy<{'riscv64': '"riscv64-linux-gnu-gcc-13 (Ubuntu 13.2.0-6ubuntu1) 13.2.0"'}>
+CONFIG_CC_VERSION_TEXT                          policy<{'riscv64': '"riscv64-unknown-linux-gnu-gcc (g2ee5e430018) 12.2.0"'}>
 CONFIG_CDNS_I3C_MASTER                          policy<{'riscv64': 'm'}>
 CONFIG_CDROM                                    policy<{'riscv64': 'y'}>
 CONFIG_CDROM_PKTCDVD                            policy<{'riscv64': 'n'}>
 CONFIG_CEC_CH7322                               policy<{'riscv64': 'm'}>
-CONFIG_CEC_CORE                                 policy<{'riscv64': 'm'}>
+CONFIG_CEC_CORE                                 policy<{'riscv64': 'y'}>
 CONFIG_CEC_GPIO                                 policy<{'riscv64': 'm'}>
 CONFIG_CEC_NOTIFIER                             policy<{'riscv64': 'y'}>
 CONFIG_CEC_PIN                                  policy<{'riscv64': 'y'}>
@@ -2079,15 +2069,15 @@ CONFIG_DEBUG_FS_ALLOW_ALL                       policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_FS_ALLOW_NONE                      policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_FS_DISALLOW_MOUNT                  policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_GPIO                               policy<{'riscv64': 'n'}>
-CONFIG_DEBUG_INFO_BTF_MODULES                   policy<{'riscv64': 'y'}>
+CONFIG_DEBUG_INFO_BTF                           policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_INFO_COMPRESSED_NONE               policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_INFO_COMPRESSED_ZLIB               policy<{'riscv64': 'n'}>
-CONFIG_DEBUG_INFO_COMPRESSED_ZSTD               policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_INFO_DWARF4                        policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_INFO_DWARF5                        policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT       policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_INFO_NONE                          policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_INFO_REDUCED                       policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO_SPLIT                         policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_IRQFLAGS                           policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_KERNEL_DC                          policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_KMEMLEAK                           policy<{'riscv64': 'n'}>
@@ -2282,10 +2272,11 @@ CONFIG_DRM_DISPLAY_CONNECTOR                    policy<{'riscv64': 'm'}>
 CONFIG_DRM_DISPLAY_DP_HELPER                    policy<{'riscv64': 'y'}>
 CONFIG_DRM_DISPLAY_HDCP_HELPER                  policy<{'riscv64': 'y'}>
 CONFIG_DRM_DISPLAY_HDMI_HELPER                  policy<{'riscv64': 'y'}>
-CONFIG_DRM_DISPLAY_HELPER                       policy<{'riscv64': 'm'}>
+CONFIG_DRM_DISPLAY_HELPER                       policy<{'riscv64': 'y'}>
 CONFIG_DRM_DP_AUX_BUS                           policy<{'riscv64': 'm'}>
 CONFIG_DRM_DP_AUX_CHARDEV                       policy<{'riscv64': 'y'}>
 CONFIG_DRM_DP_CEC                               policy<{'riscv64': 'y'}>
+CONFIG_DRM_ESWIN                                policy<{'riscv64': 'y'}>
 CONFIG_DRM_ETNAVIV                              policy<{'riscv64': 'n'}>
 CONFIG_DRM_EXEC                                 policy<{'riscv64': 'm'}>
 CONFIG_DRM_FBDEV_EMULATION                      policy<{'riscv64': 'y'}>
@@ -2302,6 +2293,8 @@ CONFIG_DRM_I2C_CH7006                           policy<{'riscv64': 'm'}>
 CONFIG_DRM_I2C_NXP_TDA9950                      policy<{'riscv64': 'm'}>
 CONFIG_DRM_I2C_NXP_TDA998X                      policy<{'riscv64': 'm'}>
 CONFIG_DRM_I2C_SIL164                           policy<{'riscv64': 'm'}>
+CONFIG_DRM_IMG                                  policy<{'riscv64': 'y'}>
+CONFIG_DRM_IMG_VOLCANIC                         policy<{'riscv64': 'm'}>
 CONFIG_DRM_ITE_IT6505                           policy<{'riscv64': 'm'}>
 CONFIG_DRM_ITE_IT66121                          policy<{'riscv64': 'm'}>
 CONFIG_DRM_KMS_HELPER                           policy<{'riscv64': 'y'}>
@@ -2671,6 +2664,9 @@ CONFIG_DW_DMAC_CORE                             policy<{'riscv64': 'm'}>
 CONFIG_DW_DMAC_PCI                              policy<{'riscv64': 'm'}>
 CONFIG_DW_EDMA                                  policy<{'riscv64': 'm'}>
 CONFIG_DW_EDMA_PCIE                             policy<{'riscv64': 'm'}>
+CONFIG_DW_HDMI_CEC                              policy<{'riscv64': 'y'}>
+CONFIG_DW_HDMI_HDCP                             policy<{'riscv64': 'n'}>
+CONFIG_DW_HDMI_I2S_AUDIO                        policy<{'riscv64': 'y'}>
 CONFIG_DW_I3C_MASTER                            policy<{'riscv64': 'm'}>
 CONFIG_DW_WATCHDOG                              policy<{'riscv64': 'm'}>
 CONFIG_DW_XDATA_PCIE                            policy<{'riscv64': 'm'}>
@@ -2747,8 +2743,12 @@ CONFIG_ERRATA_THEAD                             policy<{'riscv64': 'y'}>
 CONFIG_ERRATA_THEAD_CMO                         policy<{'riscv64': 'y'}>
 CONFIG_ERRATA_THEAD_PBMT                        policy<{'riscv64': 'y'}>
 CONFIG_ERRATA_THEAD_PMU                         policy<{'riscv64': 'y'}>
-CONFIG_ESWIN_SND_ES8388_CODEC                   policy<{'riscv64': 'y'}>
-CONFIG_ESWIN_SND_SOC_CODECS                     policy<{'riscv64': 'y'}>
+CONFIG_ESWIN_DW_HDMI                            policy<{'riscv64': 'y'}>
+CONFIG_ESWIN_MIPI_DSI                           policy<{'riscv64': 'n'}>
+CONFIG_ESWIN_MMU                                policy<{'riscv64': 'y'}>
+CONFIG_ESWIN_SND_ES8388_CODEC                   policy<{'riscv64': 'm'}>
+CONFIG_ESWIN_SND_SOC_CODECS                     policy<{'riscv64': 'm'}>
+CONFIG_ESWIN_VIRTUAL_DISPLAY                    policy<{'riscv64': 'y'}>
 CONFIG_ET131X                                   policy<{'riscv64': 'm'}>
 CONFIG_ETHERNET                                 policy<{'riscv64': 'y'}>
 CONFIG_ETHOC                                    policy<{'riscv64': 'm'}>
@@ -3000,7 +3000,7 @@ CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY       policy<{'riscv64': 'y'}>
 CONFIG_FRAMEBUFFER_CONSOLE_LEGACY_ACCELERATION  policy<{'riscv64': 'n'}>
 CONFIG_FRAMEBUFFER_CONSOLE_ROTATION             policy<{'riscv64': 'y'}>
 CONFIG_FRAME_POINTER                            policy<{'riscv64': 'y'}>
-CONFIG_FRAME_WARN                               policy<{'riscv64': '1024'}>
+CONFIG_FRAME_WARN                               policy<{'riscv64': '2048'}>
 CONFIG_FREEZER                                  policy<{'riscv64': 'y'}>
 CONFIG_FSCACHE                                  policy<{'riscv64': 'y'}>
 CONFIG_FSCACHE_DEBUG                            policy<{'riscv64': 'n'}>
@@ -3074,8 +3074,10 @@ CONFIG_GAMEPORT_FM801                           policy<{'riscv64': 'm'}>
 CONFIG_GARP                                     policy<{'riscv64': 'm'}>
 CONFIG_GCC11_NO_ARRAY_BOUNDS                    policy<{'riscv64': 'y'}>
 CONFIG_GCC_ASM_GOTO_OUTPUT_WORKAROUND           policy<{'riscv64': 'y'}>
+CONFIG_GCC_PLUGINS                              policy<{'riscv64': 'y'}>
+CONFIG_GCC_PLUGIN_LATENT_ENTROPY                policy<{'riscv64': 'n'}>
 CONFIG_GCC_SUPPORTS_DYNAMIC_FTRACE              policy<{'riscv64': 'y'}>
-CONFIG_GCC_VERSION                              policy<{'riscv64': '130200'}>
+CONFIG_GCC_VERSION                              policy<{'riscv64': '120200'}>
 CONFIG_GCOV_KERNEL                              policy<{'riscv64': 'n'}>
 CONFIG_GDB_SCRIPTS                              policy<{'riscv64': 'y'}>
 CONFIG_GEMINI_ETHERNET                          policy<{'riscv64': 'm'}>
@@ -4349,7 +4351,7 @@ CONFIG_LD_DEAD_CODE_DATA_ELIMINATION            policy<{'riscv64': 'n'}>
 CONFIG_LD_IS_BFD                                policy<{'riscv64': 'y'}>
 CONFIG_LD_ORPHAN_WARN                           policy<{'riscv64': 'y'}>
 CONFIG_LD_ORPHAN_WARN_LEVEL                     policy<{'riscv64': '"warn"'}>
-CONFIG_LD_VERSION                               policy<{'riscv64': '24100'}>
+CONFIG_LD_VERSION                               policy<{'riscv64': '24000'}>
 CONFIG_LEDS_88PM860X                            policy<{'riscv64': 'm'}>
 CONFIG_LEDS_AAT1290                             policy<{'riscv64': 'm'}>
 CONFIG_LEDS_ADP5520                             policy<{'riscv64': 'm'}>
@@ -4667,7 +4669,6 @@ CONFIG_MEDIA_ALTERA_CI                          policy<{'riscv64': 'm'}>
 CONFIG_MEDIA_ANALOG_TV_SUPPORT                  policy<{'riscv64': 'y'}>
 CONFIG_MEDIA_ATTACH                             policy<{'riscv64': 'y'}>
 CONFIG_MEDIA_CAMERA_SUPPORT                     policy<{'riscv64': 'y'}>
-CONFIG_MEDIA_CEC_RC                             policy<{'riscv64': 'y'}>
 CONFIG_MEDIA_CEC_SUPPORT                        policy<{'riscv64': 'y'}>
 CONFIG_MEDIA_COMMON_OPTIONS                     policy<{'riscv64': 'y'}>
 CONFIG_MEDIA_CONTROLLER                         policy<{'riscv64': 'y'}>
@@ -4978,7 +4979,7 @@ CONFIG_MMC35240                                 policy<{'riscv64': 'm'}>
 CONFIG_MMC_ALCOR                                policy<{'riscv64': 'm'}>
 CONFIG_MMC_BLOCK_MINORS                         policy<{'riscv64': '8'}>
 CONFIG_MMC_CB710                                policy<{'riscv64': 'm'}>
-CONFIG_MMC_CQHCI                                policy<{'riscv64': 'm'}>
+CONFIG_MMC_CQHCI                                policy<{'riscv64': 'y'}>
 CONFIG_MMC_CRYPTO                               policy<{'riscv64': 'y'}>
 CONFIG_MMC_DEBUG                                policy<{'riscv64': 'n'}>
 CONFIG_MMC_DW                                   policy<{'riscv64': 'm'}>
@@ -5002,8 +5003,8 @@ CONFIG_MMC_SDHCI_MILBEAUT                       policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_OF_ARASAN                      policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_OF_AT91                        policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_OF_DWCMSHC                     policy<{'riscv64': 'm'}>
-CONFIG_MMC_SDHCI_OF_ESWIN                       policy<{'riscv64': 'm'}>
-CONFIG_MMC_SDHCI_OF_SDIO_ESWIN                  policy<{'riscv64': 'm'}>
+CONFIG_MMC_SDHCI_OF_ESWIN                       policy<{'riscv64': 'y'}>
+CONFIG_MMC_SDHCI_OF_SDIO_ESWIN                  policy<{'riscv64': 'y'}>
 CONFIG_MMC_SDHCI_PCI                            policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_XENON                          policy<{'riscv64': 'm'}>
 CONFIG_MMC_SPI                                  policy<{'riscv64': 'y'}>
@@ -5022,7 +5023,6 @@ CONFIG_MODPROBE_PATH                            policy<{'riscv64': '"/sbin/modpr
 CONFIG_MODULES                                  policy<{'riscv64': 'y'}>
 CONFIG_MODULES_TREE_LOOKUP                      policy<{'riscv64': 'y'}>
 CONFIG_MODULES_USE_ELF_RELA                     policy<{'riscv64': 'y'}>
-CONFIG_MODULE_ALLOW_BTF_MISMATCH                policy<{'riscv64': 'n'}>
 CONFIG_MODULE_ALLOW_MISSING_NAMESPACE_IMPORTS   policy<{'riscv64': 'n'}>
 CONFIG_MODULE_COMPRESS_GZIP                     policy<{'riscv64': 'n'}>
 CONFIG_MODULE_COMPRESS_NONE                     policy<{'riscv64': 'n'}>
@@ -6002,9 +6002,8 @@ CONFIG_PAGE_REPORTING                           policy<{'riscv64': 'y'}>
 CONFIG_PAGE_SIZE_LESS_THAN_256KB                policy<{'riscv64': 'y'}>
 CONFIG_PAGE_SIZE_LESS_THAN_64KB                 policy<{'riscv64': 'y'}>
 CONFIG_PAGE_TABLE_CHECK                         policy<{'riscv64': 'n'}>
-CONFIG_PAHOLE_HAS_LANG_EXCLUDE                  policy<{'riscv64': 'y'}>
 CONFIG_PAHOLE_HAS_SPLIT_BTF                     policy<{'riscv64': 'y'}>
-CONFIG_PAHOLE_VERSION                           policy<{'riscv64': '125'}>
+CONFIG_PAHOLE_VERSION                           policy<{'riscv64': '121'}>
 CONFIG_PALMAS_GPADC                             policy<{'riscv64': 'm'}>
 CONFIG_PANEL                                    policy<{'riscv64': 'm'}>
 CONFIG_PANEL_CHANGE_MESSAGE                     policy<{'riscv64': 'n'}>
@@ -6334,7 +6333,6 @@ CONFIG_PRINTK_INDEX                             policy<{'riscv64': 'n'}>
 CONFIG_PRINTK_TIME                              policy<{'riscv64': 'y'}>
 CONFIG_PRISM2_USB                               policy<{'riscv64': 'm'}>
 CONFIG_PROBE_EVENTS                             policy<{'riscv64': 'y'}>
-CONFIG_PROBE_EVENTS_BTF_ARGS                    policy<{'riscv64': 'y'}>
 CONFIG_PROC_CHILDREN                            policy<{'riscv64': 'y'}>
 CONFIG_PROC_EVENTS                              policy<{'riscv64': 'y'}>
 CONFIG_PROC_FS                                  policy<{'riscv64': 'y'}>
@@ -6382,7 +6380,7 @@ CONFIG_PWM_ATMEL_TCB                            policy<{'riscv64': 'm'}>
 CONFIG_PWM_CLK                                  policy<{'riscv64': 'm'}>
 CONFIG_PWM_DEBUG                                policy<{'riscv64': 'n'}>
 CONFIG_PWM_DWC                                  policy<{'riscv64': 'm'}>
-CONFIG_PWM_ESWIN                                policy<{'riscv64': 'm'}>
+CONFIG_PWM_ESWIN                                policy<{'riscv64': 'y'}>
 CONFIG_PWM_FSL_FTM                              policy<{'riscv64': 'm'}>
 CONFIG_PWM_IQS620A                              policy<{'riscv64': 'm'}>
 CONFIG_PWM_LP3943                               policy<{'riscv64': 'm'}>
@@ -6466,7 +6464,9 @@ CONFIG_RAID_ATTRS                               policy<{'riscv64': 'm'}>
 CONFIG_RANDOM32_SELFTEST                        policy<{'riscv64': 'n'}>
 CONFIG_RANDOMIZE_BASE                           policy<{'riscv64': 'y'}>
 CONFIG_RANDOM_KMALLOC_CACHES                    policy<{'riscv64': 'y'}>
+CONFIG_RANDSTRUCT_FULL                          policy<{'riscv64': 'n'}>
 CONFIG_RANDSTRUCT_NONE                          policy<{'riscv64': 'y'}>
+CONFIG_RANDSTRUCT_PERFORMANCE                   policy<{'riscv64': 'n'}>
 CONFIG_RAPIDIO                                  policy<{'riscv64': 'y'}>
 CONFIG_RAPIDIO_CHMAN                            policy<{'riscv64': 'm'}>
 CONFIG_RAPIDIO_CPS_GEN2                         policy<{'riscv64': 'm'}>
@@ -7030,7 +7030,6 @@ CONFIG_RTW89_DEBUGMSG                           policy<{'riscv64': 'y'}>
 CONFIG_RTW89_PCI                                policy<{'riscv64': 'm'}>
 CONFIG_RT_MUTEXES                               policy<{'riscv64': 'y'}>
 CONFIG_RUNTIME_TESTING_MENU                     policy<{'riscv64': 'y'}>
-CONFIG_RUST_IS_AVAILABLE                        policy<{'riscv64': 'y'}>
 CONFIG_RV                                       policy<{'riscv64': 'y'}>
 CONFIG_RV_MON_WWNR                              policy<{'riscv64': 'y'}>
 CONFIG_RV_REACTORS                              policy<{'riscv64': 'y'}>
@@ -7041,7 +7040,6 @@ CONFIG_RXKAD                                    policy<{'riscv64': 'y'}>
 CONFIG_RXPERF                                   policy<{'riscv64': 'm'}>
 CONFIG_S2IO                                     policy<{'riscv64': 'm'}>
 CONFIG_SAMPLES                                  policy<{'riscv64': 'y'}>
-CONFIG_SAMPLE_AUXDISPLAY                        policy<{'riscv64': 'n'}>
 CONFIG_SAMPLE_CONFIGFS                          policy<{'riscv64': 'n'}>
 CONFIG_SAMPLE_KDB                               policy<{'riscv64': 'n'}>
 CONFIG_SAMPLE_KFIFO                             policy<{'riscv64': 'n'}>
@@ -7055,7 +7053,6 @@ CONFIG_SAMPLE_VFIO_MDEV_MBOCHS                  policy<{'riscv64': 'n'}>
 CONFIG_SAMPLE_VFIO_MDEV_MDPY                    policy<{'riscv64': 'n'}>
 CONFIG_SAMPLE_VFIO_MDEV_MDPY_FB                 policy<{'riscv64': 'n'}>
 CONFIG_SAMPLE_VFIO_MDEV_MTTY                    policy<{'riscv64': 'n'}>
-CONFIG_SAMPLE_WATCHDOG                          policy<{'riscv64': 'n'}>
 CONFIG_SATA_ACARD_AHCI                          policy<{'riscv64': 'm'}>
 CONFIG_SATA_AHCI                                policy<{'riscv64': 'm'}>
 CONFIG_SATA_DWC                                 policy<{'riscv64': 'm'}>
@@ -7903,9 +7900,8 @@ CONFIG_SND_SOC_ES7134                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_ES7241                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_ES8316                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_ES8326                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ES8328                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ES8328_I2C                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ES8328_SPI                       policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_ES8328_I2C                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ES8328_SPI                       policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_FSL_ASRC                         policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_FSL_AUDMIX                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_FSL_EASRC                        policy<{'riscv64': 'm'}>
@@ -8723,11 +8719,11 @@ CONFIG_TWL4030_WATCHDOG                         policy<{'riscv64': 'm'}>
 CONFIG_TWL6030_GPADC                            policy<{'riscv64': 'm'}>
 CONFIG_TWL6040_CORE                             policy<{'riscv64': 'y'}>
 CONFIG_TXGBE                                    policy<{'riscv64': 'm'}>
-CONFIG_TYPEC                                    policy<{'riscv64': 'y'}>
+CONFIG_TYPEC                                    policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_ANX7411                            policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_DP_ALTMODE                         policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_FUSB302                            policy<{'riscv64': 'm'}>
-CONFIG_TYPEC_FUSB303B                           policy<{'riscv64': 'y'}>
+CONFIG_TYPEC_FUSB303B                           policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_HD3SS3220                          policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_MT6360                             policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_MUX_FSA4480                        policy<{'riscv64': 'm'}>
@@ -8741,7 +8737,7 @@ CONFIG_TYPEC_STUSB160X                          policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TCPCI                              policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TCPCI_MAXIM                        policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TCPCI_MT6370                       policy<{'riscv64': 'm'}>
-CONFIG_TYPEC_TCPM                               policy<{'riscv64': 'y'}>
+CONFIG_TYPEC_TCPM                               policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TPS6598X                           policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_UCSI                               policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_WUSB3801                           policy<{'riscv64': 'm'}>
-- 
2.47.0

